sets.json

```diff
@@ -446,12 +446,12 @@
 	},
 	{
 		"code": "collector1",
```

Comment:
I think the code should roughly match the name of the set so if we're going to fix the name of these sets we should also fix the codes of `collector1` and `collector2`.

The other ones below are close to the names of the sets.